client/my-sites/backup/index.js

```diff
@@ -45,6 +46,7 @@ export default function () {
 		backupDownload,
 		wrapInSiteOffsetProvider,
 		wpcomUpsellController( WPCOMUpsellPage ),
+		showUnavailableForVaultPressSites,
```

Comment:
Middlewares execute in first-to-last order, so the most "assertive" middlewares (i.e., the ones that need to have the final say in what's rendered) go last.